src/components/OperatorPanel/OperatorPanel.jsx

```diff
@@ -10,15 +10,29 @@ import './OperatorPanel.css';
 
 const OperatorPanel = props => (
   <div className="OperatorPanel panel">
-    <header className="OperatorPanel__header">minimal chat</header>
+    <header className="OperatorPanel__header">
+      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 1026 1026" className="OperatorPanel__logo" preserveAspectRatio="xMaxYMin meet">
```

Comment:
might be worth considering moving all svg's into an `assets` or `images` or `svgs` folder and componentizing them just to clean up the code.